RNTester/js/AnimatedGratuitousApp/AnExChained.js

```diff
@@ -73,7 +73,7 @@ class AnExChained extends React.Component<Object, any> {
             <Animated.Image
               {...handlers}
               key={i}
-              source={{uri: CHAIN_IMGS[j]}}
```

Comment:
Are these changes on purpose?

Reply:
Yah, but I can remove them if you want. It just puts some images in there so you can see the effect. The images that are currently in this list are broken links.

Reply:
Sure we can keep it, I assume the images were already there?

Reply:
Yes they were.